content/administration/install/deploy.rst

```diff
@@ -355,38 +359,33 @@ of workers anymore it can not setup cron or livechat workers
 Cron Workers
 ------------
 
-To run cron jobs for an Odoo deployment as a WSGI application requires
+Starting one of the built-in Odoo servers next to the WSGI server is required to
+process cron jobs. That server must be configured to only process crons and not
+HTTP requests using the :option:`--no-http <odoo-bin --no-http>` cli option or
+the ``http_enable = False`` configuration file setting.
 
-* A classical Odoo (run via ``odoo-bin``)
-* Connected to the database in which cron jobs have to be run (via
-  :option:`odoo-bin -d`)
-* Which should not be exposed to the network. To ensure cron runners are not
-  network-accessible, it is possible to disable the built-in HTTP server
-  entirely with :option:`odoo-bin --no-http` or setting ``http_enable = False``
-  in the configuration file
+On Linux-like systems, using the multi-processing server over
+the multi-threading one is recommended to benefit from better hardware usage and increased
+stability, i.e., using the :option:`--workers=-1 <odoo-bin --workers>`
+and :option:`--max-cron-threads=n <odoo-bin --max-cron-threads>` cli options.
 
 LiveChat
 --------
 
-The second problematic subsystem for WSGI deployments is the LiveChat: where
-most HTTP connections are relatively short and quickly free up their worker
-process for the next request, LiveChat require a long-lived connection for
-each client in order to implement near-real-time notifications.
-
-This is in conflict with the process-based worker model, as it will tie
-up worker processes and prevent new users from accessing the system. However,
-those long-lived connections do very little and mostly stay parked waiting for
-notifications.
-
-The solutions to support livechat/motifications in a WSGI application are:
-
-* Deploy a threaded version of Odoo (instead of a process-based preforking
-  one) and redirect only requests to URLs starting with ``/longpolling/`` to
-  that Odoo, this is the simplest and the longpolling URL can double up as
-  the cron instance.
-* Deploy an evented Odoo via ``odoo-gevent`` and proxy requests starting
-  with ``/longpolling/`` to
-  :option:`the longpolling port <odoo-bin --longpolling-port>`.
+Using a gevent-compatible WSGI server is required for the correct operation of
+the live chat feature. That server should be able to handle many simultaneous
+long-lived HTTP requests but doesn't need a lot of processing power. All
+requests whose path starts with ``/longpolling`` should be directed to that
+server. A regular (thread/process-based) WSGI server should be used for all
+other requests.
+
+Reusing the Odoo cron server to serve the live chat requests is possible.
```

Comment:
I'm not sure that `to reuse` is the appropriate verb to use here.
As it means `to use again especially in a different way`.
But here the cron server is still used as a cron server **and** as a longpolling one.
Why not something like
```suggestion
The Odoo cron server can also be used to serve the live chat requests.
```

Reply:
suggestion applied